cmd/activator/main.go

```diff
@@ -191,6 +191,7 @@ func main() {
 	// the healthchecks or probes.
 	ah = activatorhandler.NewMetricHandler(env.PodName, ah)
 	ah = activatorhandler.NewContextHandler(ctx, ah)
+	ah = activatorhandler.NewRevisionHandler(ah)
```

Comment:
Im not convinced we need to introduce a new handler here for this PR. Let's just keep the behaviour in context_handler unmodified for now, and we can do a follow-on to discuss changing the host header parsing to accept more things.